app/Http/Controllers/DashboardController.php

```diff
@@ -16,8 +19,41 @@
 
 class DashboardController extends Controller
 {
-    public function index() : View
+    public function index(): View
     {
+        if (\Auth::user() == null)
+        {
```

Comment:
Everything within your if statement should be extracted to a private method within the controller. Also, use the `Auth::check()` method to ensure a user is logged in 😄